test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -201,7 +201,37 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
             $observed[1] | Should Be "bar"
         }
 
-        It "-File should return exit code from script: <Filename>" -TestCases @(
+        It "-File should be able to pass bool values as strings to parameters: <BoolString>" -TestCases @(
```

Comment:
We should have a test that accepts a `string` parameter and you pass `true` and `false` - we want to make sure that when we convert to `bool`, the `bool` gets properly converted back to the correct string.

And come to think about it, there is a problem - we'll lose the case that the user specified.